CONTRIBUTING.md

```diff
@@ -71,6 +124,9 @@ with minor adjustments:
 - Hard wrap at: 100
 - KDoc max comment line length: 100
 
+Remember that every method must contain a comment since it is used in the automatic 
```

Comment:
the combination of contain and just comment (not documentations comment) sound like you have to add a normal comment inside of the function body. We could link [this](https://kotlinlang.org/docs/kotlin-doc.html#kdoc-syntax).